src/Mailgun/Resource/Api/Suppressions/Complaint/ShowResponse.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Resource\Api\Suppressions\Complaint;
+
+use Mailgun\Resource\ApiResponse;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class ShowResponse implements ApiResponse
+{
+    /**
+     * @var string
+     */
+    private $address;
+
+    /**
+     * @var \DateTime
+     */
+    private $created_at;
+
+    /**
+     * @param string $address
+     * @param string $code
+     * @param string $error
```

Comment:
Wrong annotation or wrong constructor arguments :-)
Could you have a look here please?
Thanks,